src/main/java/com/dumbdogdiner/stickyapi/bukkit/item/SkullBuilder.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.item;
+
+import com.destroystokyo.paper.profile.PlayerProfile;
+import com.destroystokyo.paper.profile.ProfileProperty;
+import com.dumbdogdiner.stickyapi.common.util.StringUtil;
+import com.dumbdogdiner.stickyapi.common.util.textures.TextureHelper;
+import com.dumbdogdiner.stickyapi.common.util.textures.TextureValidator;
+import com.google.common.base.Preconditions;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.SkullMeta;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.VisibleForTesting;
+
+import java.net.URL;
+import java.util.UUID;
+
+/**
+ * Utility for constructing ItemStacks of {@link Material#PLAYER_HEAD}.
+ * <p>
+ * You can specify a group and name from the embedded texture file, or simply set a name and a texture,
+ * and generate player heads with ease!
+ * </p>
+ */
+
+@SuppressWarnings("UnusedReturnValue")
+public class SkullBuilder {
+    @Getter
+    @NotNull
+    @VisibleForTesting
+    private String category = "*";
```

Comment:
I believe Lombok does comment passthrough on `@Getter` and `@Setter` annotations, so a description of what these fields are would be appreciated <3